noxfile.py

```diff
@@ -261,6 +346,9 @@ def create_zipapp(session: Session) -> Path:
                 str(BUILD_ROOT / "requirements.windows-amd64.lock.txt"),
                 external=True,
             )
+            session.run(
```

Comment:
Noticed this bringing ARM64 up. This is accounted for ~2MB of un-needed deps for the Windows releases.